### ISSUE TITLE
False positive E1133: failure to infer type of matplotlib.rcParams['figure.figsize']

### ISSUE THREAD

**Issue by hemberger** [PyCQA/pylint#2744]
### Steps to reproduce
test.py
```python
import matplotlib
for i in matplotlib.rcParams['figure.figsize']:
    print(i)
```

`pylint -E test.py`

### Current behavior
```
************* Module test
test.py:2:9: E1133: Non-iterable value matplotlib.rcParams['figure.figsize'] is used in an iterating context (not-an-iterable)
```

### Expected behavior
No error. The `figure.figsize` param is validated to be a 2-element list by matplotlib.

### pylint --version output
```
pylint 2.2.2
astroid 2.1.0
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]
```
and also exhibited with
```
pylint 2.3.0-dev1
astroid 2.2.0-dev
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]
```

**Comment by PCManticore** [comment]
Thanks for reporting the issue. Unfortunately there's not much `pylint` can do here. `rcParams` is built at runtime and it also has heterogenous values, so there's no way for the tool to infer the exact type that `figure.figSize` actually has.
